.github/workflows/auto-publish.yml

```
@@ -0,0 +1,28 @@
+name: GitHub Book Publisher
+run-name: ${{ github.actor }} is publishing using GitHub Actions 🚀
+on:
```

Minor nit, making it YAML compliant 😄 

```suggestion
---
name: GitHub Book Publisher
run-name: ${{ github.actor }} is publishing using GitHub Actions 🚀
"on":
```